.gitmodules

```diff
@@ -9,7 +9,10 @@
 	url = https://github.com/dmlc/ps-lite
 [submodule "nnvm"]
 	path = nnvm
-	url = https://github.com/dmlc/nnvm
+	url = https://github.com/ZihengJiang/nnvm
```

Comment:
As you can see here in the log of ci(http://ec2-52-25-96-65.us-west-2.compute.amazonaws.com/blue/organizations/jenkins/mxnet/detail/PR-6345/12/pipeline/220#step-293-log-131). The size of input_shape is right in the mxnet side but wrong in the nnvm side. So I'm trying to log it out in my nnvm branch for debuging. (There maybe something wrong in CI with updated submodule). But it looks like this method doesn't work due to CI cannot change to my branch.